app/client/components/payment/update/paymentUpdated.tsx

```diff
@@ -219,7 +230,7 @@ export const ConfirmedNewPaymentDetailsRenderer = ({
                   <span
                     css={css`
                       ${valueCss};
-                      color: ${hasPaymentFailure ? news[400] : neutral[7]};
```

Comment:
little CSS bug I spotted. If user was recovering their payment method, the expiry date would show in red on confirmation page so have removed this.